code/game/dna/genes/disabilities.dm

```diff
@@ -37,8 +37,8 @@
 		M.sdisabilities|=sdisability
 	if(activation_message)
 		to_chat(M, "<span class='warning'>[activation_message]</span>")
-	else
-		testing("[name] has no activation message.")
+//	else
```

Comment:
This is actually fine, I think we want to know if a gene has no activation message, as was (is?) the case with lactose intolerance. Unless we decide that it's okay for a gene to not have these messages, but I just don't see a reason.

Reply:
🤔 perhaps this is better served by a unit test

Reply:
maybe yes